packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -780,7 +737,18 @@ const CFDPasswordModal = observer(({ form_error, platform }: TCFDPasswordModalPr
         (!has_cfd_error || is_password_error || is_password_reset);
 
     const should_show_success =
-        !has_cfd_error && is_cfd_success_dialog_enabled && is_cfd_password_modal_enabled && is_password_modal_exited;
+        !has_cfd_error &&
+        is_cfd_success_dialog_enabled &&
+        is_cfd_password_modal_enabled &&
+        is_password_modal_exited &&
+        !is_mt5_migration_modal_enabled;
+
+    const should_show_migration_success =
```

Comment:
These `!has_cfd_error`, `is_password_modal_exited` and `is_cfd_success_dialog_enabled` shared between these 2. Let's create a variable for these 3 to make the condition a bit more readable.